morpheus/_lib/include/morpheus/doca/doca_context.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#define DOCA_ALLOW_EXPERIMENTAL_API
+
+#include <doca_eth_rxq.h>
+#include <doca_flow.h>
+#include <doca_gpunetio.h>
+#include <morpheus/doca/error.hpp>
+#include <morpheus/doca/rte_context.hpp>
+
+#include <memory>
+#include <string>
+#include <type_traits>
+
+#define GPU_PAGE_SIZE (1UL << 16)
+
+namespace morpheus::doca {
+
+#pragma GCC visibility push(default)
+
+struct DocaContext
```

Comment:
Use `MORPHEUS_EXPORT` instead of the GCC visibility push/pop
```suggestion
struct MORPHEUS_EXPORT DocaContext
```